tox.ini

```diff
@@ -11,6 +11,7 @@ deps =
     pytest-asyncio
     requests
     aiohttp
+    andi
```

Comment:
I think we should remove it from here, it should be installed via setup.py. If it's not, it's an issue, and tests will correctly fail.